--- a/plugins/in_systemd/systemd_db.h
+++ b/plugins/in_systemd/systemd_db.h
@@ -43,7 +43,8 @@
     "SELECT COUNT(*) FROM in_systemd_cursor;"
 
 #define SQL_UPDATE_CURSOR                                               \
-    "UPDATE in_systemd_cursor SET cursor=@cursor, updated=@updated;"
+    "UPDATE in_systemd_cursor SET cursor=@cursor, updated=@updated "   \
+    "WHERE cursor <> @cursor;"
 
 #define SQL_DELETE_DUPS                             \
     "DELETE FROM in_systemd_cursor WHERE ROWID < "  \
